packages/patternfly-react/src/components/LoginPage/LoginPage.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
```

Comment:
Definitely a higher order component (pattern). The `LoginPageWithTranslation` and `LoginCardWithValidation` components seem to do a lot of internal state management. I think I would generally handle that in the application, but since you have done such a nice job breaking down the components, this should not be an issue (and will probably help a lot of consumers needing less fine grain detail). I'm OK proceeding here if others are ;-)

Reply:
Thank you so much @priley ! 
The consumer of this component can decided whether they 
want the component with validation or not,
by setting the `form: { validate }` prop.